easybuild/easyconfigs/f/Faiss/Faiss-1.7.2-foss-2021a-CUDA-11.3.1.eb

```
@@ -0,0 +1,44 @@
+easyblock = 'CMakeMake'
+
+name = 'Faiss'
+version = '1.7.2'
+versionsuffix = '-CUDA-%(cudaver)s'
+
+homepage = 'https://github.com/facebookresearch/faiss'
+description = "Faiss is a library for efficient similarity search and clustering of dense vectors."
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+
+source_urls = ['https://github.com/facebookresearch/faiss/archive/refs/tags/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['d49b4afd6a7a5b64f260a236ee9b2efb760edb08c33d5ea5610c2f078a5995ec']
+
+builddependencies = [
+    ('CMake', '3.20.1'),
+    ('SWIG', '4.0.2'),
+]
+
+dependencies = [
+    ('CUDA', '11.3.1', '', True),
+    ('Python', '3.9.5'),
+    ('SciPy-bundle', '2021.05'),
+]
+
+configopts = "-DFAISS_ENABLE_GPU=ON -DFAISS_ENABLE_PYTHON=ON"
```

```suggestion
configopts = "-DFAISS_ENABLE_GPU=ON -DFAISS_ENABLE_PYTHON=ON -DPython_ROOT_DIR=$EBROOTPYTHON"
```

This solves the problem I'm seeing:
```
CMake Error at /rds/projects/2017/branfosj-rse/easybuild/EL8-ice/software/CMake/3.20.1-GCCcore-10.3.0/share/cmake-3.20/Modules/FindPackageHandleStandardArgs.cmake:230 (message):
  Could NOT find Python (missing: Python_INCLUDE_DIRS Python_LIBRARIES
  Python_NumPy_INCLUDE_DIRS Development NumPy Development.Module
  Development.Embed)
Call Stack (most recent call first):
  /rds/projects/2017/branfosj-rse/easybuild/EL8-ice/software/CMake/3.20.1-GCCcore-10.3.0/share/cmake-3.20/Modules/FindPackageHandleStandardArgs.cmake:594 (_FPHSA_FAILURE_MESSAGE)
  /rds/projects/2017/branfosj-rse/easybuild/EL8-ice/software/CMake/3.20.1-GCCcore-10.3.0/share/cmake-3.20/Modules/FindPython.cmake:556 (find_package_handle_standard_args)
  faiss/python/CMakeLists.txt:118 (find_package)


-- Configuring incomplete, errors occurred!
```

I have most of the dependencies to build before I get to the rest.